src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Controllers/ApplicationsControllerTests.cs

```diff
@@ -153,5 +153,45 @@ public void POST_Application_SelectedActionIsDecline_ThrowsNotImplementedExcepti
             // Act/Assert
             Assert.ThrowsAsync<System.NotImplementedException>(() => _controller.Application(request));
         }
+
+        [Test]
+        public async Task GET_Withdrawn_ApplicationExists_ReturnsViewAndModel()
+        {
+            // Arrange
+            var request = _fixture.Create<WithdrawnRequest>();
+            _orchestrator
+                .Setup(x => x.GetWithdrawnViewModel(It.Is<WithdrawnRequest>(y => y == request)))
+                .ReturnsAsync(new WithdrawnViewModel());
+
+            // Act
+            var actionResult = await _controller.Withdrawn(request);
+            var viewResult = actionResult as ViewResult;
+            var model = viewResult.Model;
+            var withdrawnViewModel = model as WithdrawnViewModel;
+
+            // Assert
+            Assert.NotNull(actionResult);
+            Assert.NotNull(viewResult);
+            Assert.NotNull(model);
+            Assert.NotNull(withdrawnViewModel);
+        }
+
+        [Test]
+        public async Task GET_Withdrawn_ApplicationDoesntExist_ReturnsNotFound()
+        {
+            // Arrange
+            var request = _fixture.Create<WithdrawnRequest>();
+            _orchestrator
+                .Setup(x => x.GetWithdrawnViewModel(It.Is<WithdrawnRequest>(y => y == request)))
+                .ReturnsAsync((WithdrawnViewModel)null);
+
+            // Act
+            var actionResult = await _controller.Withdrawn(request);
```

Comment:
again just cast to the type after the method call, and have the one assert, cleaner

Reply:
I personally don't see the issue with multiple `Assert`s - in my opinion this would allow for quicker diagnosis of an introduced bug, if there ever was one.